pkg/osquery/extension.go

```diff
@@ -164,6 +170,10 @@ func NewExtension(client service.KolideService, db *bbolt.DB, opts ExtensionOpts
 		return nil, errors.Wrap(err, "creating DB buckets")
 	}
 
+	if err := SetupLauncherKeys(db); err != nil {
+		return nil, errors.Wrap(err, "setting up initial launcher keys")
```

Comment:
nit / opinion: I feel like we should go ahead and switch to using `fmt.Errorf()` for all new error wrapping. I don't think the inconsistency in the file is really a problem.

Reply:
Reasonable. I think I got these. Sometime we'll just need to change a bunch en-mass.